src/main/java/spring/boot/parser/service/CsvParserService.java

```diff
@@ -0,0 +1,8 @@
+package spring.boot.parser.service;
+
+import com.univocity.parsers.common.record.Record;
+import java.util.List;
+
+public interface CsvParserService {
+    List<Record> getRecords();
```

Comment:
add a test class for this service please

Reply:
Added, but coverage not full.